x-pack/plugins/task_manager/server/saved_objects/model_versions/task_model_versions.ts

```diff
@@ -20,4 +20,18 @@ export const taskModelVersions: SavedObjectsModelVersionMap = {
       create: taskSchemaV1,
     },
   },
+  '2': {
+    changes: [
+      {
+        type: 'mappings_addition',
+        addedMappings: {
+          partition: { type: 'integer' },
+        },
+      },
+    ],
+    schemas: {
+      forwardCompatibility: taskSchemaV2.extends({}, { unknowns: 'ignore' }),
```

Comment:
should we add this to the v1 schema as well?

Reply:
Added in this commit [5a5d188](https://github.com/elastic/kibana/pull/188001/commits/5a5d188d4c00f41c2300bad864730311d81ccd24)